pkg/node/ingress_endpoint.go

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package node
+
+const (
+	templateIngressEndpointID = uint64(0xfffe)
+)
+
+var (
+	ingressEndpointID = templateIngressEndpointID
```

Comment:
Let's avoid adding more globals here, as they make modularisation and testing hard. 

Take a look at `local_node_store.go` and it's `LocalNode` struct to see how to structure the code in a way that avoids globals; ideally the ingress endpoint functionality directly depends on and interacts with the `LocalNodeStore`, without going through some `pkg/node` function.

Reply:
Added a commit to remove the global `node.endpointID` that I had used as the pattern for the `node.ingressEndpointID`, then changed the ingress endpoint ID use the same pattern.

Reply:
@bimmlerd After Jussi's pushback on the approach I took for the above I noticed that the node package getter I added was not actually used at all... so I removed the setter and the global variable as well.

The already existing host endpoint ID getter is used to feed it into bpf datapath; it is currently based on a global variable in the node package.